sql/mob_skill_lists.sql

```diff
@@ -2350,22 +2355,25 @@ INSERT INTO `mob_skill_lists` VALUES ('Effigy_Shield',705,1077);
 INSERT INTO `mob_skill_lists` VALUES ('Effigy_Shield',705,1078);
 INSERT INTO `mob_skill_lists` VALUES ('Effigy_Shield',705,1079);
 INSERT INTO `mob_skill_lists` VALUES ('Effigy_Shield',705,1080);
-INSERT INTO `mob_skill_lists` VALUES ('Cerebrator',706,1243);
-INSERT INTO `mob_skill_lists` VALUES ('Cerebrator',706,1244);
-INSERT INTO `mob_skill_lists` VALUES ('Cerebrator',706,1245);
-INSERT INTO `mob_skill_lists` VALUES ('Cerebrator',706,1248);
-INSERT INTO `mob_skill_lists` VALUES ('Cerebrator',706,1251);
-INSERT INTO `mob_skill_lists` VALUES ('Cerebrator',706,1252);
-INSERT INTO `mob_skill_lists` VALUES ('Delver',707,1231);
-INSERT INTO `mob_skill_lists` VALUES ('Delver',707,1232);
-INSERT INTO `mob_skill_lists` VALUES ('Delver',707,1233);
-INSERT INTO `mob_skill_lists` VALUES ('Delver',707,1234);
-INSERT INTO `mob_skill_lists` VALUES ('Delver',707,1274);
-INSERT INTO `mob_skill_lists` VALUES ('Propagator',708,741);
-INSERT INTO `mob_skill_lists` VALUES ('Propagator',708,744);
-INSERT INTO `mob_skill_lists` VALUES ('Propagator',708,747);
-INSERT INTO `mob_skill_lists` VALUES ('Propagator',708,750);
-INSERT INTO `mob_skill_lists` VALUES ('Propagator',708,752);
+INSERT INTO `mob_skill_lists` VALUES ('Thinker_NM',706,1243);
+INSERT INTO `mob_skill_lists` VALUES ('Thinker_NM',706,1244);
+INSERT INTO `mob_skill_lists` VALUES ('Thinker_NM',706,1245);
+INSERT INTO `mob_skill_lists` VALUES ('Thinker_NM',706,1248);
+INSERT INTO `mob_skill_lists` VALUES ('Thinker_NM',706,1251);
+INSERT INTO `mob_skill_lists` VALUES ('Thinker_NM',706,1252);
+INSERT INTO `mob_skill_lists` VALUES ('Thinker_NM',706,1271);
+INSERT INTO `mob_skill_lists` VALUES ('Craver_NM',707,1229);
+INSERT INTO `mob_skill_lists` VALUES ('Craver_NM',707,1232);
+INSERT INTO `mob_skill_lists` VALUES ('Craver_NM',707,1233);
+INSERT INTO `mob_skill_lists` VALUES ('Craver_NM',707,1234);
+INSERT INTO `mob_skill_lists` VALUES ('Craver_NM',707,1274);
+INSERT INTO `mob_skill_lists` VALUES ('Craver_NM',707,1275);
+INSERT INTO `mob_skill_lists` VALUES ('Craver_NM',707,1276);
+-- INSERT INTO `mob_skill_lists` VALUES ('Propagator',708,741); -- Replaced with list 138
```

Comment:
If this isn't needed anymore, it can be left as a single line

Reply:
I left it open for further use. There are un-implemented Gorger type mobs that may have skill sets that are slightly different.